### DISCUSSION TITLE
Option to store password

### DISCUSSION THREAD

**Discussion by ntbanks924** [muesli/obs-cli#18] [Ideas]
Hello @muesli,
I've been using this tool in tandem with the streamdeck-ui application in Pop OS. I'm wondering if there is a way to add the value of the --password flag to a variable in some config file instead of adding it to the command in obs-cli? I think this feature would be very helpful.
Thanks,
Nick

**Comment by muesli** [comment]
This idea is now tracked as #28.

**Comment by onyx-and-iris** [comment]
Hi. I've added ability to load connection info from config file here 

https://github.com/onyx-and-iris/obs-cli-v5

I would make PR but this fork includes the updates by yusefnapora so it's only working with websocket v5.

**Reply by Milliw** [reply]
Wow! Thanks for that one!
Works like a charm for me =)
